python/lsst/ts/cbp/csc.py

```diff
@@ -31,6 +31,8 @@ class CBPCSC(salobj.ConfigurableCsc):
     component : `CBPComponent`
     simulator : `None` or `MockServer`
     telemetry_task : `asyncio.Future`
+    telemetry_time : `float`
+    in_position_time : `int`
```

Comment:
Please document what this is, and consider renaming it to `in_position_timeout`.

Also (I should have mentioned this earlier) please consider renaming "telemetry_time" to "telemetry_interval".

"time" is a bit overloaded. I try to always name time limits "timeout" because that matches the usage in `asyncio` and try to reserve `time` for absolute times.